### PR TITLE
[CI][VSTS] Bring back the API diff from stable

### PR DESCRIPTION
**Pull request by mandel-macaque**
Bring back the status of the API diff for PRs and commits. The comment will look similar to [this](https://github.com/xamarin/xamarin-macios/commit/816b11f3c981189872caaaffae471959e31104df#commitcomment-47300105) with the diff that a message has been added and a title for the pkgs to make the message to have a nicer format.

API diff can be accessed via vsdrops for team members or via gists. The [gist](https://gist.github.com/vs-mobiletools-engineering-service2/82268da7cd3786afa79a9d2e8baef5bb) is an example of what a user will see.

### CODE REVIEW

File: tools/devops/automation/templates/build/upload-vsdrops.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# Upload the API diff to vsdrops to be able to locate it
+- task: ms-vscs-artifact.build-tasks.artifactDropTask-1.artifactDropTask@0
+  displayName: 'Publish API diff to Artifact Services Drop'
+  inputs:
+    dropServiceURI: 'https://devdiv.artifacts.visualstudio.com/DefaultCollection'
+    dropMetadataContainerName: 'DropMetadata-APIDiff'
+    buildNumber: 'xamarin-macios/device-tests/$(Build.BuildNumber)/$(Build.BuildId)/APIDiff'
+    sourcePath: $(STABLE_APIDDIFF_PATH)
+    detailedLog: true
+    usePat: true
```
Comment (rolfbjarne) [inline tools/devops/automation/templates/build/upload-vsdrops.yml:38]: 👀

**Review by rolfbjarne — APPROVED** [review]

File: tools/devops/automation/templates/build/upload-vsdrops.yml (hunk 1 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/build/upload-vsdrops.yml]: hahahaha ok, I updated by editor settings.

**Review by spouliot — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**